### PR TITLE
Adds display property for demo configurations

### PR DESCRIPTION
**Pull request by GeorgePlukov**

### CODE REVIEW

File: pages/assets/demo-resizable-preview.js [hunk 1]
```diff
@@ -218,7 +223,7 @@ class ComponentCatalogDemoResizablePreview extends LitElement {
 				}
 				.layout {
 					align-items: ${align};
-					display: flex;
+					display: ${display};
 					flex-wrap: wrap;
```
Comment (margaree) [inline pages/assets/demo-resizable-preview.js:227]: I guess it'd probably be a bit messy to not have the `flex` related properties unless display is set to flex.

**Review by margaree — APPROVED** [review]
Can you also add this to the documentation https://github.com/BrightspaceUI/documentation/blob/main/docs/adding-component.md#code-blocks ?

File: pages/assets/demo-resizable-preview.js (hunk 1 above)
Comment (GeorgePlukov) [reply pages/assets/demo-resizable-preview.js:227]: Should be able to do this without making it too confusing, Ive got something in the works

**Review by margaree — APPROVED** [review]
Looks good!

**Pull request merged**